ci-operator/step-registry/ipi/conf/aws/blackholenetwork/ipi-conf-aws-blackholenetwork-ref.yaml

```diff
@@ -1,9 +1,6 @@
 ref:
   as: ipi-conf-aws-blackholenetwork
-  from_image:
-    namespace: ocp
-    name: "4.5"
```

Comment:
13cec2ff43263 (#9791) explains the history of `aws` availability in early 4.y upi-installer images.  If we want to drop this pin here, we'd be breaking this step for 4.4, etc.  Which we're maybe ok with now?  But if so, we should call that out explicitly in the commit message, so folks realize we're intentionally breaking the use case.

Reply:
I changed back to use a pinned-version (4.12) upi-installer image for the step.